js/compiler/compiler_test.go

```diff
@@ -114,14 +102,14 @@ func TestCompile(t *testing.T) {
 
 		t.Run("Invalid", func(t *testing.T) {
 			src := `1+(function() { return 2; )()`
-			_, _, err := c.Compile(src, "script.js", "", "", true)
+			_, _, err := c.Compile(src, "script.js", "", "", true, CompatibilityModeES6)
 			assert.IsType(t, &goja.Exception{}, err)
 			assert.Contains(t, err.Error(), `SyntaxError: script.js: Unexpected token (1:26)
 > 1 | 1+(function() { return 2; )()`)
```

Comment:
This test was lying, well, still does. :) It's part of the "ES5" suite, yet this error is returned by Babel, not Goja (which returns a less descriptive `script.js: Line 1:27 Unexpected token ) (and 2 more errors)`).

Should I just update the assert to reflect that? Right now I'm passing `CompatibilityModeES6` to make it pass.